src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -691,6 +715,23 @@ private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operatio
             else
             {
                 CheckOperationAttributes(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, symbol);
+                if (symbol is IMethodSymbol method)
+                {
+                    if (method.IsGenericMethod)
+                    {
+                        foreach (var typeArgument in method.TypeArguments)
```

Comment:
Do these type arguments also include containing type's type arguments?

Reply:
Good question I would assume it only lists immediate type arguments, i will check

Reply:
> @buyaa-n I think you want to target this PR to `release\5.0.2xx` branch?

That must be right cc @jeffhandley

Reply:
It was not, added that scenario too